### DISCUSSION TITLE
AvaloniaLocator.Current.GetRequiredService<IWindowingPlatform>,what's the instead

### DISCUSSION THREAD

**Discussion by Cohn21** [AvaloniaUI/Avalonia#12788] [Q&A]
in earlier verison,use  var platform = AvaloniaLocator.Current.GetRequiredService<IWindowingPlatform>(); get IWindowingPlatform,
now in V11, AvaloniaLocator.Current.GetService can no longer be used,how to get IWindowingPlatform

**Comment by timunie** [comment]
What do you need it for?

**Reply by Cohn21** [reply]
want to get all screens resolution

**Reply by timunie** [reply]
Window.Screens should do the trick.
